src/app/jscaip/AlignementMinimax.ts

```diff
@@ -1,59 +0,0 @@
-import { MGPOptional } from '../utils/MGPOptional';
-import { Coord } from './Coord';
-import { Minimax } from './Minimax';
-import { Move } from './Move';
-import { BoardValue } from './BoardValue';
-import { SCORE } from './SCORE';
-import { GameState } from './GameState';
-
-export interface BoardInfo {
-    status: SCORE,
-    victory: MGPOptional<Coord[]>,
-    preVictory: MGPOptional<Coord>,
-    sum: number,
-}
-export abstract class AlignementMinimax<M extends Move,
```

Comment:
Alignment d'ailleurs ?

Reply:
ouais si il revient ça sera sous le nom AlignmentMinimax. Mais m'es d'avis que cette classe venait juste généraliser le concept de boucler sur le plateau, donc son utilité était maigre, pour preuve elle n'était utilisée qu'à un endroit, dans le minimax le plus sale :p

Reply:
Alignment Minimax étant utilisé pour Six, sa logique est plus générique que N-In-A-Row-Helper, au pire ce dernier est une alternative simplifiée, mais pour Six, Teeko, et les jeux ou des formes linéaire (ou non) de pièce à toi forment une victoire, pourrons réutiliser (Teeko Advanced pas plus tard que dans pas longtemps, pourra) et non je trouve ça plus élégant de déjà mettre ça à sa place, les petites classes sont mieux !

Reply:
à faire dans le ticket-à-créer qui généralise cela ?

Reply:
Nooon #SaveAlignmentMinimax #NotInMyName #AllCodesMatters